scipy/stats/tests/test_distributions.py

```diff
@@ -2668,6 +2668,36 @@ def test_minimum_data_zero_gh17801(self):
             _assert_less_or_close_loglike(dist, data, dist.nnlf)
 
 
+class TestPowerNorm:
+
+    # survival function references were computed with mpmath via
+    # from mpmath import mp
+    # float(mp.ncdf(-x)**c)
+
+    @pytest.mark.parametrize("x, c, ref",
+                             [(9, 1, 1.1285884059538405e-19),
+                              (20, 2, 7.582445786569958e-178)])
+    def test_sf(self, x, c, ref):
+        assert_allclose(stats.powernorm.sf(x, c), ref)
+
+    # inverse survival function references were computed with mpmath via
+    # from mpmath import mp
+    # def isf_mp(q, c):
+    #     arg = q**(1 / c)
+    #     return float(- mp.sqrt(2) * mp.erfinv(2 * arg -1))
```

Comment:
I know it's "just a comment", but...
```suggestion
    #     return float(-mp.sqrt(2) * mp.erfinv(2*arg - 1))
```